learn/on-premise/aws/install-prerequisites.md

```diff
@@ -103,15 +103,15 @@ systemctl start docker
 yum install python3 -y
 ```
 
-### Extra configurations on RHEL External Instances
+### Extra configurations on RHEL Studio Workspace Instance/App Deployment Instance
```

Comment:
Not ok with this change. Lets talk and arrive at a better way to explain external instances.

why not just call is a "instance"/"host computer"